js/buttonConnection.js

```diff
@@ -0,0 +1,13 @@
+var currencyChoiceContainer = document.getElementById('currency-choice-container');
+var currentCurrencyButton = document.getElementById('current-currency');
+
+currencyChoiceContainer.onclick = function(e){
+  // var target = e.target;
+  if(usd.checked){
+    currentCurrencyButton.style.backgroundImage="url('img/usd_checked.svg')";
+  } else if(euro.checked){
+    currentCurrencyButton.style.backgroundImage="url('img/euro_checked.svg')";
+  } else if (rrubles.checked) {
+    currentCurrencyButton.style.backgroundImage="url('img/rruble_checked.svg')";
+  }
+};
```

Comment:
This one is bad one :) 

Here is a couple of questions and mistakes 

* What is `usd.checked`, `euro.checked`, `rrubles.checked` Where do you define it ?
* It's bad practice to mix representation layer and logic layer, you have to move button decoration into `CSS` and use proper css class when change button style 

For instance you probably will have 3 different classes like 
```css
.uds {
}
.eur {
}
.rub {
} 
```

BTW It's always better to use `ISO` names for currencies and resources like this, it will help you in future to automate this kind of actions